src/pages/_document.tsx

```diff
@@ -6,6 +6,7 @@ const Document = (): ReactElement => {
 		<Html>
 			<Head>
 				<link rel="stylesheet" href="/flowplayer/style/flowplayer.css" />
+				<link rel="shortcut icon" href="/favicon.ico" />
```

Comment:
Moet je deze hier nog expliciet zetten? Ik dacht dat je gwn de file kon aanpassen en dat Next de favicon zelf oppikt

Reply:
Blijkbaar niet
ik dacht mss doordat we een custom document voorzien dat hij het niet ging door hebben, maar next is slim :p